Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM ubuntu:trusty
 MAINTAINER "Syncano DevOps Team" <devops@syncano.com>
 
 ENV LAST_REFRESHED 2016-03-25
+ENV SYNCANO_APIROOT https://api.syncano.io/
+
+RUN groupadd -r syncano && \
+    useradd -u 1000 -r -g syncano syncano -d /tmp -s /bin/bash && \
+    mkdir /home/syncano && \
+    chown -R syncano /home/syncano
+
+# enable everyone to use /tmp
+RUN chmod 1777 /tmp
+# -- CUT --
```

Comment:
Yeah, but I just thought that we could make it explicit with CUT BEGIN here and CUT END at USER.
About: ENV NODE_PATH /home/syncano/v0.4/node_modules -> we can move this higher above.